docs/resources/Channel.md

```diff
@@ -700,6 +705,8 @@ This endpoint supports requests with `Content-Type`s of both `application/json`
 > info
 > Note that when sending `application/json` you must send at least one of `content` or `embed`, and when sending `multipart/form-data`, you must send at least one of `content`, `embed` or `file`. For a `file` attachment, the `Content-Disposition` subpart header MUST contain a `filename` parameter.
```

Comment:
Would we be able to create a message with only a `message_reference` param here or would we also need to include at least one of `content`, `embed` and/or `file`?

Reply:
Wouldn't just quoting a message and providing no other content be pointless behaviour outside producing spam, or is there a valid use case for wanting to be able to do this?

Reply:
Well I sent this before [https://github.com/discord/discord-api-docs/pull/2118#issuecomment-703005795](this comment below), I was going off [above comment](https://github.com/discord/discord-api-docs/pull/2118#issuecomment-702594111) which seems to be incorrect? Either way clarification would be nice now that i've asked.

Reply:
Regarding the comments, the one below is a screenshot of the feature in action, and the one above is incorrect. Quoting will be replaced (not markdown removed however), but it's not nearly the same thing. When replying to a message it auto fills into a "@user ", it's pretty much the same content field as sending a normal message. So if I had to make an educated guess some sort of message content will likely be required, while yes quoting doesn't require it, that's because the quote itself counts as message content.

Reply:
A message is still required to have a `content`, `embed`, or `file` regardless of `message_reference`.